servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -402,11 +407,8 @@ protected Hash doOps(
     }
   }
 
-  private static CommitMeta meta(Principal principal, CommitMeta commitMeta) {
-    if (commitMeta.getCommitter() != null) {
```

Comment:
why remove this check? Was it moved elsewhere?

Reply:
Yes, at line 394

We're re-using this method for merge/transplant, but the check is specific to the new commits.